ejector/ejection_transactor.go

```diff
@@ -22,3 +31,223 @@ type EjectionTransactor interface {
 	// Complete the ejection proceedings against the operator with the given address.
 	CompleteEjection(ctx context.Context, addressToEject gethcommon.Address) error
 }
+
+var _ EjectionTransactor = &ejectionTransactor{}
+
+// ejectionTransactor is the production implementation of the EjectionTransactor interface.
+type ejectionTransactor struct {
```

Comment:
Your docs don't mention thread safety- would be good to explicitly state for all structs